src/Avalonia.Controls/Control.cs

```diff
@@ -216,6 +216,12 @@ public class Control : InputElement, IDataTemplateHost, INamed, IVisualBrushInit
         /// <inheritdoc/>
         bool IDataTemplateHost.IsDataTemplatesInitialized => _dataTemplates != null;
 
+        public Control()
+        {
+            // Add a default HoldGestureRecognizer
+            GestureRecognizers.Add(new HoldGestureRecognizer());
```

Comment:
1) how would that work with recognizers explicitly set from xaml? how one removes default recognizer from xaml?
2) this adds quite a bit of memory footprint for each control

Reply:
1. It's appended to the collection, so the default recognizer isn't replaced

Reply:
It's a good point, if we want to make holding gesture opt-in instead of opt-out. So, by default it can be enabled only on some controls in the XAML template, but not everywhere.

Reply:
But we also can make it a simple gesture like Tapped or DoubleTapped without any IGestureRecognizer.